morphic.js

```diff
@@ -9403,16 +9439,34 @@ MenuItemMorph.prototype.mouseEnter = function () {
         this.image = this.highlightImage;
         this.changed();
     }
+    if (this.doubleClickAction instanceof MenuMorph) {
+        var w = this.parentThatIsA(WorldMorph);
```

Comment:
Right you are.  I had seen this.world used for the world in MenuMorph, so I tried that and of course it didn't work in MenuItemMorph, and I hadn't yet discovered this.world(), so I fell back on something I was confident about.  Fixing...